src/plot_api/plot_api.js

```diff
@@ -750,6 +751,17 @@ function cleanData(data, existingData) {
             }
         }
 
+        // fix typo in colorscale definition
```

Comment:
we don't want YIGnBu and YIOrBu to be considered valid colorscale names, we just want to clean up an old typo.